src/json_to_stix/transformers.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+from . import json_to_stix
+from datetime import datetime, timezone
+import pprint
+import logging
+logging.basicConfig(format='%(asctime)s %(message)s', level=logging.DEBUG)
+
+pp = pprint.PrettyPrinter(indent=2, width=50)
```

Comment:
Do we even use this variable and the above pretty print? I can't see where it gets referenced.

Reply:
We don't. I'll remove it